homeassistant/components/zwave_js/device_condition.py

```diff
@@ -0,0 +1,225 @@
+"""Provide the device conditions for Z-Wave JS."""
+from __future__ import annotations
+
+from typing import cast
+
+import voluptuous as vol
+from zwave_js_server.const import CommandClass, ConfigurationValueType
+from zwave_js_server.model.value import ConfigurationValue, get_value_id
+
+from homeassistant.const import CONF_CONDITION, CONF_DEVICE_ID, CONF_DOMAIN, CONF_TYPE
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers import condition, config_validation as cv
+from homeassistant.helpers.config_validation import DEVICE_CONDITION_BASE_SCHEMA
+from homeassistant.helpers.typing import ConfigType, TemplateVarsType
+
+from . import DOMAIN
+from .const import (
+    ATTR_COMMAND_CLASS,
+    ATTR_ENDPOINT,
+    ATTR_PROPERTY,
+    ATTR_PROPERTY_KEY,
+    ATTR_VALUE,
+)
+from .helpers import async_get_node_from_device_id
+
+CONF_SUBTYPE = "subtype"
+
+CONF_VALUE_ID = "value_id"
+
+NODE_STATUS_TYPES = {"asleep", "awake", "dead", "alive"}
+CONFIG_PARAMETER_TYPE = "config_parameter"
+VALUE_TYPE = "value"
+CONDITION_TYPES = {*NODE_STATUS_TYPES, CONFIG_PARAMETER_TYPE, VALUE_TYPE}
```

Comment:
Generally we've kept the device automations limited to simple use cases. For more extensive use cases people can use normal state conditions etc. Are all these types really what we expect non-advanced users to use? 

I also see that all node status types get their own condition. That seems overkill. Why not just have a single state and a dropdown to pick the state?

Reply:
we can do a single condition for this. The reason I added this is because we disable the sensor entity by default